src/rtcModule/webrtc.cpp

```diff
@@ -782,7 +783,13 @@ void RtcModule::removeCall(Id chatid, bool retry)
         {
             if (retry || itHandler->second->callParticipants())
             {
-                itHandler->second->removeAllParticipants();
+                bool reconnectionState = false;
+                if (mRetryCall.find(chatid) != mRetryCall.end())
+                {
+                    reconnectionState = true;
+                }
+
+                itHandler->second->removeAllParticipants(reconnectionState);
```

Comment:
Next time, please, try to simply do:
```suggestion
                itHandler->second->removeAllParticipants(mRetryCall.find(chatid) != mRetryCall.end());
```